spec/Section 3 -- Type System.md

```diff
@@ -837,7 +837,6 @@ input ExampleInputObject {
 Original Value          | Variables       | Coerced Value
 ----------------------- | --------------- | -----------------------------------
 `{ a: "abc", b: 123 }`  | {null}          | `{ a: "abc", b: 123 }`
-`{ a: 123, b: "123" }`  | {null}          | `{ a: "123", b: 123 }`
```

Comment:
Instead of removing it, can you update it to show that this produces an `Error`?